Library/Homebrew/test/cask/conflicts_with_spec.rb

```diff
@@ -1,25 +0,0 @@
-# frozen_string_literal: true
-
-RSpec.describe "conflicts_with", :cask do
```

Comment:
It seemed weird to have this in its own file – I moved this to `dsl_spec`, where there are similar tests.